WWDC 2016 ScreenSaver/WWDC2016ScreenSaver.swift

```diff
@@ -38,7 +38,7 @@ class WWDC2016ScreenSaverView: ScreenSaverView {
     let words = [":", ";", "\\", "/", ".", "!", "?",
                  "+", "-", "*", "&", "^", "[", "]",
                  "(", ")", "#", "@", "&", "<", ">",
-                 "~"]
+                 "~", " "/*, "do", "let", "var", "try", "if", "for", "as"*/]
```

Comment:
It'd be cool to have those multi char words in there as well, but there would be more logic necessary, so I didn't yet implement that